CoreGraphFramework/src/au/gov/asd/tac/constellation/graph/undo/UndoGraphEditOperation.java

```diff
@@ -716,7 +716,7 @@ public void updateUndo(UndoGraphEditState state, int instruction) {
         @Override
         public void execute(UndoGraphEditState state, GraphWriteMethods graph) {
             if (VERBOSE) {
-                System.out.println("setDoubleValueOperation.execute(" + state.getCurrentAttribute() + ", " + state.getCurrentId() + ", " + Double.longBitsToDouble((Double.doubleToRawLongBits(graph.getDoubleValue(state.getCurrentAttribute(), state.getCurrentId())) ^ state.getCurrentDouble())) + ")");
+                System.out.println("setDoubleValueOperation.execute(" + state.getCurrentAttribute() + ", " + state.getCurrentId() + ", " + Double.longBitsToDouble((long) (Double.doubleToRawLongBits(graph.getDoubleValue(state.getCurrentAttribute(), state.getCurrentId())) ^ state.getCurrentDouble())) + ")");
```

Comment:
I think there were a couple instances of this in the lines below that also need to be reverted (casting a long that is)